frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -95,6 +102,7 @@ export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProp
     consolePage,
     nicsPage,
     disksPage,
+    ...(isSnapshotsAvailable ? [snapshotsPage] : []),
```

Comment:
are we expecting more pages? IMO simple if would be more readable.

Reply:
this is a one line and it is a convention.
check https://github.com/openshift/console/blob/master/frontend/webpack.config.ts#L192 for example

Reply:
I wouldn't cite webpack.config.ts as a convention. I know there are both use cases in the code base.

This one is less elegant. 
Do as you want - I can live with both.